browser/resources/settings/brave_overrides/site_details.ts

```diff
@@ -151,26 +145,6 @@ RegisterPolymerTemplateModifications({
     const usageSection = templateContent.querySelector('div#usage')
     if (!usageSection) {
       console.error(`[Brave Settings Overrides] Couldn't find usageSection item`)
```

Comment:
I think we can delete this code as well, as it's no longer used from what I can see.